testing/api/test_api.py

```diff
@@ -505,78 +980,20 @@ def test_delete_device_no_mac(empty_devices_dir, testrun): # pylint: disable=W06
 
   device_1.pop("mac_addr")
 
-  # Test that device_1 can't delete with no mac address
-  r = requests.delete(f"{API}/device/",
-                      data=json.dumps(device_1),
-                      timeout=5)
-  assert r.status_code == 400
-  assert len(local_get_devices()) == 1
-
-
-# Currently not working due to blocking during monitoring period
-@pytest.mark.skip()
-def test_delete_device_testrun_running(testing_devices, testrun): # pylint: disable=W0613
-
-  payload = {"device": {"mac_addr": BASELINE_MAC_ADDR, "firmware": "asd"}}
-  r = requests.post(f"{API}/system/start", data=json.dumps(payload), timeout=10)
-  assert r.status_code == 200
-
-  until_true(
-      lambda: query_system_status().lower() == "waiting for device",
-      "system status is `waiting for device`",
-      30,
-  )
-
-  start_test_device("x123", BASELINE_MAC_ADDR)
-
-  until_true(
-      lambda: query_system_status().lower() == "in progress",
-      "system status is `in progress`",
-      600,
-  )
-
-  device_1 = {
-        "manufacturer": "Google",
-        "model": "First",
-        "mac_addr": BASELINE_MAC_ADDR,
-        "test_modules": {
-            "dns": {"enabled": True},
-            "connection": {"enabled": True},
-            "ntp": {"enabled": True},
-            "baseline": {"enabled": True},
-            "nmap": {"enabled": True},
-        },
-    }
-  r = requests.delete(f"{API}/device/",
-                      data=json.dumps(device_1),
-                      timeout=5)
-  assert r.status_code == 403
-
-
-def test_start_testrun_started_successfully(
-    testing_devices, # pylint: disable=W0613
-    testrun): # pylint: disable=W0613
-  payload = {"device": {
-             "mac_addr": BASELINE_MAC_ADDR,
-             "firmware": "asd", 
-             "test_modules": {
-                "dns": {"enabled": False},
-                "connection": {"enabled": True},
-                "ntp": {"enabled": False},
-                "baseline": {"enabled": False},
-                "nmap": {"enabled": False}
-            }}}
-  r = requests.post(f"{API}/system/start", data=json.dumps(payload), timeout=10)
-  assert r.status_code == 200
-
+  # Test that device_1 can't delete with no mac address
+  r = requests.delete(f"{API}/device/",
+                      data=json.dumps(device_1),
+                      timeout=5)
+  assert r.status_code == 400
+  assert len(local_get_devices()) == 1
 
 # Currently not working due to blocking during monitoring period
 @pytest.mark.skip()
```

Comment:
This should be able to be enabled. It does not go to the monitor state so should pass.